packages/react-canvas-core/src/CanvasEngine.ts

```diff
@@ -122,6 +144,9 @@ export class CanvasEngine<
 			});
 		};
 
+		// if repaintDebounce is set to true, debounce the 'repaint' function by the milliseconds defined by repaintDebounceMs
+		const repaintFn = repaintDebounce ? debounce(repaint, repaintDebounceMs) : repaint;
```

Comment:
also I would prefer not using ternary operators outside of react :) Prettier should actually remove this, did you run `yarn pretty` ?

Reply:
@dylanvorster Yes I did, it doesn't remove the ternary operator. 
Anyway I'll remove it but why is that?
There's a precise reason?